scripts/typescript_checks.py

```diff
@@ -57,13 +810,105 @@ def validate_compiled_js_dir() -> None:
             'in %s: %s' % (COMPILED_JS_DIR, TSCONFIG_FILEPATH, out_dir))
 
 
+def compile_strict_tsconfig(
+    config_path: str, error_messages: List[str]) -> None:
```

Comment:
Please indent this 4 more spaces so the arguments aren't at the same level as the function code

Reply:
Done